specifyweb/specify/tree_views.py

```diff
@@ -359,6 +360,65 @@ def tree_rank_item_count(request, tree, rankid):
     count = tree_rank_count(tree, rank.id)
     return HttpResponse(toJson(count), content_type='application/json')
 
+TAXON_TREES = ["Mineral", "Rock", "Meteorite", "Fossil"]
+TAXON_RANKS = ["One", "Two", "Three"]
+
+# TODO: Add openapi schema for this endpoint
+@tree_mutation
+def add_geo_default_trees(request):
+    for tree in TAXON_TREES:
+        if Taxontreedef.objects.filter(name=tree).exists():
+            continue
+        tree_name = f"{tree} Taxon"
+        ttd = Taxontreedef.objects.create(name=tree_name)
+        ttd.save()
+        rank_id = 0
+        ttdi = None
+        for rank in TAXON_RANKS:
+            name = f"{tree} {rank}"
+            rank_id += 10
```

Comment:
the lowest rank is of rankid 10 then?? No, it should be 0.

Reply:
I know that is how Specify 6 did it.  I think it is more robust to not make the root rank ID 0, that why there is room to add a new root rank if the user needs it.  This way is more robust, if the root is made to be 0, then the user will not have this option.  Just my opinion, if we really don't care about this user-case, then I'm ok changing it.

Reply:
The root should be 0 though?

Reply:
Trees work when when the root is not zero.  For example, see this unit test
https://github.com/specify/specify7/blob/080b32ceaec14d747291bbb020d6329f48dcedc0/specifyweb/specify/tests/test_trees.py#L215-L227

Reply:
No. what you mean is a unit tests work. NOT THE ENTIRE APP. Search for usages of rankid, and all the places historically which could have depend on it, and then we will talk

See https://github.com/specify/specify7/blob/080b32ceaec14d747291bbb020d6329f48dcedc0/specifyweb/frontend/js_src/lib/components/PickLists/TreeLevelPickList.tsx#L30

Reply:
Sorry, could've sworn the tree viewer worked in non-zero root ids. I'll edit the code.  We might want to change that in the future, though the use-case is probably pretty rare anyway.